src/boot/efi/efi.h

```diff
@@ -441,6 +441,7 @@ typedef struct {
         } *ConfigurationTable;
 } EFI_SYSTEM_TABLE;
 
+extern EFI_HANDLE IMG;
```

Comment:
not a fan of global variables. but i guess if this is inherently a per-process (well to the level that efi knows "process") then maybe its ok